### DISCUSSION TITLE
Where is nodejs and npm installed?

### DISCUSSION THREAD

**Discussion by mfgabriel92** [coreybutler/nvm-windows#1129] [Support] [answered]
Hi,

I installed the latest lts with `nvm install lts` on a terminal ran with admin rights, but the commands `node` and `npm` are not recognizable, and I cannot find where they are installed for me to add to Path.

Thank you.

**Comment by dmduggan** [comment]
You should use `nvm use latest` to configure the installed version.

**Reply by coreybutler** [reply, marked as answer]
@dmduggan is correct, either `nvm use lts` or `nvm use latest` (or `nvm use x.x.x` where x.x.x is the lts version).

The node versions are installed in the `NVM_HOME` directory.

**Reply by mfgabriel92** [reply]
The `use` did the trick. Thanks!
